aws/scripts/sh/website_installation.sh

```
@@ -2,8 +2,9 @@
 echo #### Install Software
 n install "${NODEJS_VERSION:?Node.js version is not set}"
 n "${NODEJS_VERSION:?Node.js version is not set}"
-git clone -b "$WEBSITE_GIT_REPOSITORY_BRANCH" "$WEBSITE_GIT_REPOSITORY_LINK.git" /codebuild-user/website
-cd /codebuild-user/website/ || {
+mkdir -p $CODEBUILD_SRC_DIR/website
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)